GameDB: some title fixes letter A + SCAJ-30004 + SLPM-62346

rufotheone-fr opened the pull request:
### Description of Changes
Some title fixes.

### Rationale behind Changes
GameDB accuracy.

### Suggested Testing Steps
...

rufotheone-fr commented:
BTW, is there a process to add unlicensed titles?
For example: [Game Studio Version 1.03a](http://redump.org/disc/80140/)

rufotheone-fr commented:
Is my PR missing something that prevent it to be merged?

Tokman5 commented on bin/resources/GameIndex.yaml:
```diff
@@ -27570,7 +27570,7 @@ SLPM-65576:
   region: "NTSC-J"
 SLPM-65579:
-  name: "After School Love Beat"
+  name: "Simple 2000 Series Ultimate Vol. 27 - Houkago no Love Beat"
```
SLPM-65579 is the original version of ``Houkago no Love Beat``, not the one of Simple series.

rufotheone-fr replied:
Both Houkago no Love Beat and Akai Ito Simple/SuperLite 2000 versions got a new serial number on their boxes but had the same disc as original versions so both are fine.

Tokman5 commented on bin/resources/GameIndex.yaml:
```diff
@@ -28072,7 +28072,7 @@ SLPM-65731:
   region: "NTSC-J"
 SLPM-65732:
-  name: "Akaiito"
+  name: "SuperLite 2000 Vol. 34 - Akai Ito"
```
SLPM-65732 is the original version of ``Akai Ito``

rufotheone-fr replied:
Reverted to base title

JordanTheToaster commented:
Use the proper naming for commits please eg "GameDB: X"

drachenschanze commented:
Hyper Sports 2002 Winter (Japan) 

![Capture](https://user-images.githubusercontent.com/96800307/187210694-39e4f170-5333-4f73-b4d3-a2590e6d7420.PNG)

JordanTheToaster commented on bin/resources/GameIndex.yaml:
```diff
@@ -37266,7 +37266,7 @@ SLPS-25828:
   region: "NTSC-J"
 SLPS-25829:
-  name: "A.C.E. Another Century's Episode 2 [Special Vocal Version]"
+  name: "Another Century's Episode 2 [Special Vocal Version]"
```
I suggest just not touching it until the others can be confirmed the name is correct as it is in the db so please leave it.

rufotheone-fr replied:
@JordanTheToaster : I've forgotten to properly name my commits, sorry... Will do that henceforth.
About Another Century's Episode, every box displays `A.C.E. Another Century's Episode 1/2/3` but every spine only got 'Another Century's Episode` in katakana. When an acronym is used in the japanese title, it should be written on the spine with romajis or katakanas.
But, we have the 2 choices I was talking about (add A.C.E. in every title or remove it).

@drachenschanze Sorry, I don't get the point of your comment?

Tokman5 replied:
Looking around some Japanese pages, I think A.C.E. should be removed from every title. They don't seem to consider ``A.C.E.`` as a part of full title name.

drachenschanze commented:
Curious George (Asia)

![Capture](https://user-images.githubusercontent.com/96800307/187325747-648ddda7-9f67-4c22-96f5-c17698d8a450.PNG)


http://redump.org/disc/63786/

drachenschanze commented:
Kan Wo Long Xian Shen Wei (Asia)

![image](https://user-images.githubusercontent.com/96800307/187343861-a010f65d-e1ef-46d6-ba5f-f012959e917b.png)

http://redump.org/disc/26167/

F0bes commented:
Just an unrelated side note, it's not recommended to push directly to your downstream forks `master` branch. Unless your intent is to provide a different release of PCSX2, it's advised to simply keep your `master` branch in sync with ours, and then branch off of your master when you want to PR a change upstream. :)

rufotheone-fr commented:
> Just an unrelated side note, it's not recommended to push directly to your downstream forks `master` branch. Unless your intent is to provide a different release of PCSX2, it's advised to simply keep your `master` branch in sync with ours, and then branch off of your master when you want to PR a change upstream. :)

Mrlinkwii already told me to create a branch then push from there but my master branch was outdated and when I'm trying to sync it with upstream, it wants to create a PR o_O I'm trying to figure out how Github Desktop works but ATM I'm pushing PR, wait for it to be merged then I delete my fork and recreate one u_u How do you keep your master branch synced?

rufotheone-fr commented:
> Kan Wo Long Xian Shen Wei

I've added this into my PR

drachenschanze commented:
Keiei Simulation - Jurassic Park (Japan) 

![Capture](https://user-images.githubusercontent.com/96800307/187381958-4616bb53-943a-4c19-928b-7e5c8acb2afa.PNG)


http://redump.org/disc/40783/

JordanTheToaster commented:
> Keiei Simulation - Jurassic Park (Japan)
> 
> ![Capture](https://user-images.githubusercontent.com/96800307/187381958-4616bb53-943a-4c19-928b-7e5c8acb2afa.PNG)
> 
> http://redump.org/disc/40783/

Rather than just keep posting here everything you seem to find just make an issue with a list.

rufotheone-fr commented:
> Keiei Simulation - Jurassic Park (Japan)
> 
> ![Capture](https://user-images.githubusercontent.com/96800307/187381958-4616bb53-943a-4c19-928b-7e5c8acb2afa.PNG)
> 
> http://redump.org/disc/40783/

Added this one too, squashed the commits, properly named it and now?

F0bes commented:
> > Just an unrelated side note, it's not recommended to push directly to your downstream forks `master` branch. Unless your intent is to provide a different release of PCSX2, it's advised to simply keep your `master` branch in sync with ours, and then branch off of your master when you want to PR a change upstream. :)
> 
> Mrlinkwii already told me to create a branch then push from there but my master branch was outdated and when I'm trying to sync it with upstream, it wants to create a PR o_O I'm trying to figure out how Github Desktop works but ATM I'm pushing PR, wait for it to be merged then I delete my fork and recreate one u_u How do you keep your master branch synced?

Sorry, I don't personally use GitHub desktop, as it's missing crucial features. In this case you probably needed to hard reset your forks master branch. Since GitHub desktop doesn't provide any of that functionality, you would've had to run
`git checkout master`
`git reset --hard origin/master`
This would've cleared any difference between your master and PCSX2's master.

---
Don't do this if you have any PRs **open** pointing to commits on your master branch, as it will remove the commit and might cause issues.

rufotheone-fr commented:
I'm waiting for this PR to be merged then I'll delete my master branch/recreate one and create a branch for future PR.

So the process is:
- pushing PR from my sub-branch
- wait for it to be merged
(- delete the sub-branch?)
- resync master branch
(- create a new sub-branch and repeat?)

refractionpcsx2 commented on bin/resources/GameIndex.yaml:
```diff
@@ -12851,7 +12851,7 @@ SLES-51893:
   region: "PAL-E"
 SLES-51896:
-  name: "Gallop Racer"
+  name: "Attheraces Presents  Gallop Racer"
```
Double space after Presents

refractionpcsx2 commented:
Thanks :)

rufotheone-fr commented:
Thank you too, more coming :)
